Node 10 compatibility issue in libxml-mt

As noted here: https://github.com/gagern/libxmljs/issues/12
I too cant get past node 9 in : 
'''
    %> npm install gulp-xslt --save-dev

    > libxmljs-mt@0.18.0 install ./node_modules/libxmljs-mt
    > node-gyp rebuild

  CXX(target) Release/obj.target/xmljs/src/libxmljs.o
../src/libxmljs.cc:69:17: error: no member named 'IsDead' in 'v8::V8'
    if (v8::V8::IsDead())
        ~~~~~~~~^
1 error generated.
make: *** [Release/obj.target/xmljs/src/libxmljs.o] Error 1
'''

libxmljs-mt seems stale.

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,9 +8,10 @@ addons:
     packages:
     - g++-4.8
 node_js:
-  - "4"
   - "5"
   - "6"
   - "7"
   - "8"
+  - "9"
+  - "10"
   - node

--- a/index.js
+++ b/index.js
@@ -38,7 +38,7 @@ module.exports = function(template, config) {
             try {
                 var document = libxslt.libxmljs.parseXml(file.contents);
                 var output = stylesheet.apply(document, parameters, {outputFormat: 'string', noWrapParams: true});
-                file.contents = new Buffer(output);
+                file.contents = Buffer.from(output);
             } catch (e) {
                 return throwError(e.message);
             }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "gulp-xslt",
-  "version": "4.0.0",
+  "version": "5.0.0",
   "description": "XSLT transformation plugin for gulp",
   "main": "index.js",
   "keywords": [
@@ -13,8 +13,8 @@
     "url": "git://github.com/vecmezoni/gulp-xslt.git"
   },
   "dependencies": {
-    "event-stream": "~3.3.4",
-    "libxslt": "~0.7.0",
+    "event-stream": "~4.0.1",
+    "libxslt": "git+https://github.com/alexdee2007/node-libxslt.git",
     "plugin-error": "~1.0.1"
   },
   "author": "Alexander Inozemtsev <alexander.inozemtsev@gmail.com>",
@@ -29,14 +29,14 @@
   },
   "devDependencies": {
     "gulp": "~4.0.0",
-    "libxml-xsd": "~0.5.0",
+    "libxml-xsd": "git+https://github.com/vecmezoni/node-libxml-xsd.git",
     "mocha": "*",
     "should": "*",
     "stream-assert": "~2.0.2"
   },
   "engineStrict": true,
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 5"
   },
   "bugs": {
     "url": "https://github.com/vecmezoni/gulp-xslt/issues"

